content/en/tracing/guide/instrument_custom_method.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Instrument a custom method to get deep visibility into your business logic
+kind: guide
+disable_toc: true
+further_reading:
+- link: "/tracing/guide/alert_anomalies_p99_database/"
+  tag: "3 mins"
+  text: "Alert on anomalous p99 latency of a database service"
+- link: "tracing/guide/week_over_week_p50_comparison/"
+  tag: "2 mins"
+  text: "Compare a service’s latency to the previous week"
+- link: "/tracing/guide/slowest_request_daily/"
+  tag: "3 mins"
+  text: "Debug the slowest trace on the slowest endpoint of a web service"
+- link: "tracing/guide/"
+  tag: ""
+  text: "All guides"
+---
+_8 minutes to complete_
+
+{{< img src="tracing/guide/custom_span/custom_span_1.png" alt="Analytics View" responsive="true" style="width:90%;">}}
+
+In order to provide you deep visibility into your business logic, Datadog APM allows you to customize the spans that make up your traces based on your needs and implementation. This empowers you to trace any method in your codebase and even specific components within methods. You can use this to optimize and monitor critical areas of your application at the granularity that works for you.
+
+Custom spans are useful when you have methods that perform multiple actions like fetching data from a payments database, sanitizing, processing and verifying the response, and finally returning the results to a customer.
```

Comment:
This may be a bit confusing. If I read that quickly, it seems that you need to add custom spans for "fetching data from a payments database, sanitizing, processing and verifying the response, and finally returning the results to a customer" that is actually what we instrument already out of the box.

It's described better in the next block where it's important to define that while the processing is a custom business logic and it's already traced, it doesn't have a granularity you want and so why you need custom instrumentation. I think you can just merge saying "we auto-instrument x and y, but for z you can use custom spans to increase the visibility".

Reply:
See next